extensions/wikia/FlowTracking/FlowTracking.hooks.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+class FlowTrackingHooks {
+	const CREATE_PAGE_UNRECOGNIZED_FLOW = 'create-page-unrecognized';
+
+	public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameters "$out, $skin". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)